var/spack/repos/builtin/packages/sbp/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sbp(CMakePackage):
+    """Library providing an API for Piksi GNSS devices."""
+
+    homepage = 'https://github.com/swift-nav/libsbp'
+    git      = 'https://github.com/swift-nav/libsbp'
+
+    maintainers = ['jayvdb']
+
+    version('3.4.10', tag='v3.4.10', submodules=True)
```

Comment:
Can we download a tarball of the sources? Asking since that permits to have a sha256 checksum, while tags can be moved around.

Reply:
If we do tarballs, we have the problem of fetching the correct version of the [`c/cmake/common`](https://github.com/swift-nav/libsbp/tree/master/c/cmake) submodule.  I didnt find other packages using tarballs with submodules, so I wasnt sure whether that would be acceptable.  It is a bit messy, as the git submodule sha isnt in the tarball.

Also there are sbp releases about every month, so the git approach makes it a bit simpler to add updates here as part of the release process.

fwiw, we dont move tags.  We push a new release, and/or yank releases from package repos if a serious problem was found.

Reply:
Fair enough then. Thanks for the reply.